README.md

```diff
@@ -1,44 +1,154 @@
+
+# Steps
+
+1. [Clone the repo](#1-clone-the-repo)
+2. [Create IBM Cloud services](#2-create-compose-for-mongodb-service-with-ibm-cloud)
+3. [Build your images](#3-build-your-images)
+4. [Configure Deployment files](#4-configure-deployment-files)
+5. [Deploy the application](#5-deploy-the-application)
+6. [Search for a product in Amazon](#6-search-for-a-product-in-amazon)
+
+### 1. Clone the repo
+
+```
+$ git clone https://github.com/IBM/watson-review-analyzer
+$ cd watson-review-analyzer/
+```
+
+### 2. Create IBM Cloud services
+
+Create the following service:
 
 * [**Watson Discovery**](https://console.ng.bluemix.net/catalog/services/discovery)
+* [**IBM Cloud Kubernetes**](https://console.bluemix.net/containers-kubernetes/catalog/cluster)
 
-* [**Cloudant NoSQL DB**](https://console.bluemix.net/catalog/services/cloudant-nosql-db)
+### 3. Build your images
 
+The Node.js app will be packaged into a Docker image. This will be used by the Kubernetes Cluster
 
-## 2. Installation
+```
+$ docker build -t DOCKERHUB_USERNAME/watson-review-analyzer:1.0 .
+$ docker push DOCKERHUB_USERNAME/watson-review-analyzer:1.0
+```
 
-Edit the `config.json.sample` file to include your Watson Discovery username and password. Then rename the file to `config.json`.
+### 4. Configure Deployment files
+> Make sure you have set-up `kubectl` to use your cluster.
 
-Next, install all the required dependencies for the project.
+Add your Watson Discovery credentials in `config.json.sample` and rename it to `config.json`:
 
+![Discovery Credentials](docs/discovery-credentials.png)
+
+`config.json`:
 ```
-$ npm install
+{
+  "discoveryUsername" : "YOUR_WATSON_DISCOVERY_USERNAME",
+  "discoveryPassword" : "YOUR_WATSON_DISCOVERY_PASSWORD"
+}
 ```
 
-Next, provide your Cloudant URL from your Cloudant Service Credentials as an environment variable.
+Create configmap in Kubernetes for `config.json`
 
 ```
-export CLOUDANT_URL=https://1111405-438f-4374-9b43-975e0220f3cd-bluemix:935684a91478c5f8918d3d0b2e004ef6fe825939ed2304f28ec2225fa9a952df@8bb72405-438f-4374-9b43-975e0220f3cd-bluemix.cloudant.com
+$ kubectl create configmap watson-discovery-config --from-file=config.json
 ```
-Now, you are ready to run the app!
+
+### 5. Deploy the application
+<!--Deploy in kubernetes (1) nodejs app and (2) couchdb instance. Access via external ip-->
+
+Deploy the CouchDB database. This is where the reviews and Watson Discovery results will be stored.
+
+```
+$ kubectl apply -f couchdb.yaml
+```
+
+Deploy the Watson Review Analyzer app.
+
 ```
-$ node app.js
+$ kubectl apply -f watson-review-analyzer.yaml
 ```
 
-## 3. Usage
-`curl http://localhost:8080/reviews/[product-id]`
+You can access the application in the browser via the Load Balancer's IP (External IP).
+If you don't have the Load Balancer enabled, you can access it through one of your Kubernetes' worker IP and service Node port of watson-review-analyzer.
+
+<pre>
```

Comment:
wrap the code block in three backticks instead of the HTML `<pre>` tag

<pre>
```
code foo
```
</pre>